Switch/Switch.module.less

```diff
@@ -36,6 +40,11 @@
 			.icon {
 				-webkit-transition: left 0.2s, color 0.2s;
```

Comment:
We don't need `-webkit-` prefix since the build tool will do for us. Please remove these.